TestLib/Exception/GREYFailureFormatter.m

```diff
@@ -23,24 +23,6 @@
 
 @implementation GREYFailureFormatter
 
-+ (NSString *)formatFailureForError:(GREYError *)error
```

Comment:
Can you send a separate PR for this?

Reply:
I had to change the method parameter to not be variadic in order for FailureFormatter to work with the rest of the changes. And when i changed that parameter (to not be variadic), it was then the exact same method as the other method, so i had to remove it. So, I think this removal should be in this PR.